bin/wp-cli.php

```diff
@@ -128,8 +165,13 @@ public function flush() {
 	 *      # Index six specific posts
 	 *      wp searchpress index 12340 12341 12342 12343 12344 12345
 	 *
+	 *      # Index posts published between 11-1-2015 and 12-15-2015 inclusive
+	 *      wp searchpress index --from=11-01-2015 --to=12-15-2015
```

Comment:
See my previous comment that this should be in mysql date format, `YYYY-MM-DD`, to be consistent with other WP-CLI commands (as well as everything else in WP)